values.yaml

```diff
@@ -20,6 +13,10 @@ podSecurityPolicyName: ""
 # resolution to work correctly.
 clusterDomainSuffix: ".svc.cluster.local"
 
+# imagePullPolicy -- Sets the policy for pulling a container image across all
+# services.
+imagePullPolicy: Always
```

Comment:
In the spirit of prefixing everything, should we make a separate imagePullPolicy key for each service + environments?

Reply:
This will happen for Coder 2.0. I'm not sure if workspace providers will release before that, but if not this is just for dogfood.